Meta/Lagom/Tools/CodeGenerators/IPCCompiler/main.cpp

```diff
@@ -795,12 +796,17 @@ void build(StringBuilder& builder, Vector<Endpoint> const& endpoints)
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    if (arguments.argc != 2) {
-        outln("usage: {} <IPC endpoint definition file>", arguments.strings[0]);
-        return 1;
-    }
+    StringView ipc_file;
+    StringView output_file = "-"sv;
+
+    Core::ArgsParser parser;
+    parser.add_positional_argument(ipc_file, "IPC endpoint definition file", "input");
+    parser.add_option(output_file, "Place to write file", "output", 'o', "output-file");
```

Comment:
should the cmake build use this option?

Reply:
good point. it's a one line change anyway. added